src/math.rs

```diff
@@ -42,102 +42,163 @@ impl IterationData<Math, ConstantFold> for IterData {
 // operators from FPCore
 define_language! {
     pub enum Math {
-        // special FP constants
+
+        // FPCore constants
         "TRUE" = True,
         "FALSE" = False,
-        "E" = E,
         "LOG2E" = Log2E,
         "LOG10E" = Log10E,
         "LN2" = Ln2,
         "LN10" = Ln10,
-        "PI" = Pi,
         "PI_2" = Pi2,
         "PI_4" = Pi4,
         "1_PI" = Pi1Alt,
         "2_PI" = Pi2Alt,
         "2_SQRTPI" = Sqrtpi2,
         "SQRT2" = Sqrt2,
         "SQRT1_2" = Sqrt1_2,
```

Comment:
Should we delete the rest of these too?

Reply:
If Herbie has no intention of using these constants, it makes sense to remove them.

Reply:
Agreed.